lib/coinbase/wallet.rb

```diff
@@ -36,28 +47,69 @@ def import(data)
       end
 
       # Creates a new Wallet on the specified Network and generate a default address for it.
+      # @param interval_seconds [Integer] The interval at which to poll the CDPService for the Wallet to
+      # have an active seed, if using a ServerSigner, in seconds
+      # @param timeout_seconds [Integer] The maximum amount of time to wait for the ServerSigner to
+      # create a seed for the Wallet, in seconds
       # @param network_id [String] (Optional) the ID of the blockchain network. Defaults to 'base-sepolia'.
+      # @param server_signer [Boolean] (Optional) whether Wallet should use project's server signer. Defaults to false.
       # @return [Coinbase::Wallet] the new Wallet
-      def create(network_id: 'base-sepolia')
+      def create(network_id: 'base-sepolia', interval_seconds: 0.2, timeout_seconds: 20)
```

Comment:
These arguments are pretty odd for a `create` call. In a `wait!` call it makes sense that `interval_seconds` is the wait interval.

But in the create call it is not immediately obvious what that entails.

If we truly need to block in this call (Which I would advocate against if at all possible) I would consider calling this `wait_interval_seconds` 

Wallet's will themselves have a status whether they're "active" or not at some point to better encapsulate this.

Reply:
@alex-stone we want to block the creation for now until signer is active. is it ok to followup after v0.0.6 release with a `wait!`?